reference/mongodb/bson/javascript/construct.xml

```diff
@@ -12,7 +12,7 @@
   <methodsynopsis>
    <modifier>final</modifier> <modifier>public</modifier> <methodname>MongoDB\BSON\Javascript::__construct</methodname>
    <methodparam><type>string</type><parameter>code</parameter></methodparam>
-   <methodparam choice="opt"><type class="union"><type>array</type><type>object</type></type><parameter>scope</parameter></methodparam>
+   <methodparam choice="opt"><type class="union"><type>array</type><type>object</type><type>null</type></type><parameter>scope</parameter><initializer>&null;</initializer></methodparam>
```

Comment:
Is `<initializer>` relevant if we already have `<methodparam choice="opt">`? I'm not sure if there's prior art in other extension docs, but if that's too much trouble to find I have no objection to keeping this as-is since it's technically correct.

Reply:
Without an initialiser, this shows the following method synopsis in the docs:
```php
final public MongoDB\BSON\Javascript::__construct(string $code, array|object $scope = ?)
```

The initialiser replaces the question mark with an actual default value, which IMO makes more sense.